src/renderer/entities/wallet/lib/permission-utils.ts

```diff
@@ -142,3 +143,16 @@ function canVote(wallet: Wallet): boolean {
 
   return true;
 }
+
+function canDelegate(wallet: Wallet): boolean {
+  if (walletUtils.isWatchOnly(wallet)) return false;
+  if (walletUtils.isProxied(wallet)) {
+    const isAnyProxy = accountUtils.isAnyProxyType(wallet.accounts[0]);
+    const isGovernanceProxy = accountUtils.isGovernanceProxyType(wallet.accounts[0]);
+    const isNonTransferProxy = accountUtils.isNonTransferProxyType(wallet.accounts[0]);
+
+    return isAnyProxy || isGovernanceProxy || isNonTransferProxy;
+  }
+
+  return true;
+}
```

Comment:
Basically, we can use this permission not only for delegation but for all governance operations (vote, unlock) 
Maybe have it as a common function?  Like `canGovernanceOperate` - can't come up with a better name now but you got the point